docker/README.md

```diff
@@ -33,6 +28,61 @@ In order to run service locally you need to configure hosts. Add below line to `
 127.0.0.1	wikia-local.com dev.wikia-local.com muppet.dev.wikia-local.com
 ```
 
+## Deploy Pipeline
+
+This section describe how to depoly mediawiki to various environments, how they work and how to introduce changes to it.
+
+There are 2 main parts of all deployments:
+1. Jenkins job that is defined in jenkins-jobs reposiotory that accepts needed parameters to deploy to a given environment
+2. a jenkins file that is in this repo that is being executed by Jenkins as a pipeline job
+
+### How to deploy to sandbox
+
+1. Open [mediaiwki-deploy-sandbox](http://jenkins.wikia-prod:8080/blue/organizations/jenkins/mediawiki-deploy-sandbox/activity)
+2. Press run button, fill in the form and submit the form
+3. Profit!
+
+#### Backstage
+- jenkins file: https://github.com/Wikia/app/blob/dev/docker/sandbox/Jenkinsfile
+
+
+### How to deploy to preview
+
+This job is ran by [daily-release-cut](http://jenkins.wikia-prod:8080/blue/organizations/jenkins/daily-release-cut/activity) but if you need to deploy manually:
+
+1. Open [mediaiwki-deploy-preview](http://jenkins.wikia-prod:8080/blue/organizations/jenkins/mediawiki-deploy-preview/activity)
```

Comment:
What do you think about adding warning e.g. "Please keep in my that preview deployment will automatically start all functional tests" or similar?